### PR TITLE
Pupil Invisible LSL Relay

### PR DESCRIPTION
**Pull request by romanroibu**

### CODE REVIEW

File: pupil_invisible_lsl_relay/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Pupil Invisible Gaze Relay
+
+## Installation
+
+```bash
+git clone https://github.com/romanroibu/App-PupilLabs.git
```
Comment (cboulay) [inline pupil_invisible_lsl_relay/README.md]: This should be updated for the pull request.

**Comment by cboulay** [comment]
Hi @romanroibu , can you give a brief description of what's changing? There have been users in the past who were a bit blind-sided by the previous major change to the pupil LSL plugin dataformat; it would be good if we could provide an explanation. What's the Pupil Invisible Gaze Relay device monitor? What are the advantageous for this vs. the pupil_capture method? etc.

Thank you!

**Comment by papr** [comment]
This is an additional lsl app. The old one does not change. Only thing that should change is the instructions

**Review by papr — CHANGES_REQUESTED** [review]
@romanroibu Please add a top-level readme explaining the main difference between the two lsl relays and their intended usages (Pupil Core + Capture, Pupil Invisible + stand-alone). Also point to the respective readme files in the subfolders for more information.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
```
Comment (papr) [inline setup.py]: `find_packages` is unused

**Comment by gmierz** [comment]
I'm not a reviewer here, but I want to make a friendly request to add a relay version control number to this similar to what was done here: https://github.com/labstreaminglayer/App-PupilLabs/pull/7

Please, and thank you! :)

File: README.md [hunk 3]
```diff
@@ -1,48 +1,15 @@
-# Pupil LSL Relay Plugin
+# App-PupilLabs
 
-Plugin for _[Pupil Capture](https://github.com/pupil-labs/pupil/wiki/Pupil-Capture)_ that publishes realtime gaze data using the [lab streaming layer](https://github.com/sccn/labstreaminglayer) framework.
+This repository contains implementations of relays that publish realtime gaze data using the [lab streaming layer](https://github.com/sccn/labstreaminglayer) framework.
 
-## Installation
+- **`pupil_capture`** contains a [Pupil Capture][pupil-capture-app] plugin that works with the [Pupil Core headset][pupil-core-headset].
```
Comment (papr) [inline README.md]:
```suggestion
- **`pupil_capture`** contains a [Pupil Capture][pupil-capture-app] plugin that works with the [Pupil Core headset][pupil-core-headset] and the [VR/AR add-ons][vr-ar-addons].
```

File: README.md [hunk 4]
```diff
@@ -1,48 +1,15 @@
...
-## LSL Clock Synchronization
-
-The `Pupil LSL Relay` plugin adjusts Capture's timebase to synchronize Capture's own clock with the `pylsl.local_clock()`. This allows the recording of native Capture timestamps and removes the necessity of manually synchronize timestamps after the effect.
-
-**Warning**: The time synchronization will potentially break if other time alternating actors (e.g. the `Time Sync` plugin or `hmd-eyes`) are active. 
+[pupil-capture-app]: https://docs.pupil-labs.com/core
+[pupil-core-headset]: https://docs.pupil-labs.com/core/hardware
+[pupil-invisible-headset-and-app]: https://docs.pupil-labs.com/invisible
+[pupil-core-lsl-readme]: https://github.com/labstreaminglayer/App-PupilLabs/blob/master/pupil_capture/README.md
+[pupil-invisible-lsl-readme]: https://github.com/labstreaminglayer/App-PupilLabs/blob/master/pupil_invisible_lsl_relay/README.md
```
Comment (papr) [inline README.md:15]:
```suggestion
[pupil-invisible-lsl-readme]: https://github.com/labstreaminglayer/App-PupilLabs/blob/master/pupil_invisible_lsl_relay/README.md
[vr-ar-addons]: https://pupil-labs.com/products/vr-ar/
```

File: README.md [hunk 5]
```diff
@@ -1,48 +1,15 @@
...
-'diameterright_2d' (right/left): Pupil diameter in pixels
-
-'diameterright_3d' (right/left): Pupil diameter in mm
-
-## LSL Clock Synchronization
-
-The `Pupil LSL Relay` plugin adjusts Capture's timebase to synchronize Capture's own clock with the `pylsl.local_clock()`. This allows the recording of native Capture timestamps and removes the necessity of manually synchronize timestamps after the effect.
-
-**Warning**: The time synchronization will potentially break if other time alternating actors (e.g. the `Time Sync` plugin or `hmd-eyes`) are active. 
+[pupil-capture-app]: https://docs.pupil-labs.com/core
```
Comment (papr) [inline README.md]:
```suggestion
[pupil-capture-app]: https://github.com/pupil-labs/pupil/releases/latest
```

File: README.md [hunk 6]
```diff
@@ -1,48 +1,15 @@
...
+[pupil-core-headset]: https://docs.pupil-labs.com/core/hardware
```
Comment (papr) [inline README.md]:
```suggestion
[pupil-core-headset]: https://pupil-labs.com/products/core
```

File: README.md [hunk 7]
```diff
@@ -1,48 +1,15 @@
...
+[pupil-invisible-headset-and-app]: https://docs.pupil-labs.com/invisible
```
Comment (papr) [inline README.md]:
```suggestion
[pupil-invisible-headset-and-app]: https://pupil-labs.com/products/invisible/
```

File: README.md [hunk 8]
```diff
@@ -1,48 +1,15 @@
-# Pupil LSL Relay Plugin
+# App-PupilLabs
 
-Plugin for _[Pupil Capture](https://github.com/pupil-labs/pupil/wiki/Pupil-Capture)_ that publishes realtime gaze data using the [lab streaming layer](https://github.com/sccn/labstreaminglayer) framework.
+This repository contains implementations of relays that publish realtime gaze data using the [lab streaming layer](https://github.com/sccn/labstreaminglayer) framework.
 
-## Installation
+- **`pupil_capture`** contains a [Pupil Capture][pupil-capture-app] plugin that works with the [Pupil Core headset][pupil-core-headset].  
+More information on how to install and use the plugin is available [here][pupil-core-lsl-readme]
+- **`pupil_invisible_lsl_relay`** is a command line tool for publishing data from the [Pupil Invisible headset][pupil-invisible-headset-and-app].
```
Comment (papr) [inline README.md]:
```suggestion
- **`pupil_invisible_lsl_relay`** is a command line tool for publishing data from [Pupil Invisible][pupil-invisible-headset-and-app].  
```

**Review by papr — CHANGES_REQUESTED** [review]
I think it would be better to link to the product pages instead.

**Review by papr — APPROVED** [review]

**Comment by papr** [comment]
@cboulay please also have a look at the release notes draft

**Pull request merged**